.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -21,7 +21,8 @@ https://github.com/strapi/strapi/blob/main/CONTRIBUTING.md#reporting-an-issue
 ### Required System information
 
 <!-- Please ensure you are using the Node LTS version (v16 or v18 or v20) -->
-<!-- Strapi v3 is not supported unless it is a critical/high security issue -->
```

Comment:
Do we want to keep this but change it to "is not supported." ?

Reply:
```suggestion
<!-- Strapi v3 is no longer supported, please update to Strapi v4 -->
```

Reply:
damn you github can't commit on deleted lines